scripts/ci/pylint_todo.txt

```diff
@@ -107,6 +107,7 @@
 ./airflow/contrib/hooks/sagemaker_hook.py
 ./airflow/contrib/hooks/salesforce_hook.py
 ./airflow/contrib/hooks/segment_hook.py
+./airflow/contrib/hooks/sentry_hook.py
```

Comment:
New files should be fixed and not ignored.  This list contains only files that are not compatible when we introduce the pylint. The files from this list are gradually deleted.